rotkehlchen/tests/api/test_user_assets.py

```diff
@@ -33,7 +34,7 @@
 
 @pytest.mark.parametrize('use_clean_caching_directory', [True])
 @pytest.mark.parametrize('start_with_logged_in_user', [True])
-def test_adding_custom_assets(rotkehlchen_api_server, globaldb):
+def test_add_user_assets(rotkehlchen_api_server, globaldb):
```

Comment:
can you also change docstrings here in this and the other test functions you did not do it? The name is a good change, thank you.